subscription-service/src/test/resources/ngsild/subscription_with_time_interval.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
don't forget to remove this file, you're no longer using it

Reply:
I do it